protected/humhub/modules/admin/controllers/UserController.php

```diff
@@ -150,6 +160,26 @@ public function actionEdit()
             ];
         }
 
+        // Change Password Form
+        if ($canEditAdminFields) {
+            $definition['elements']['Password'] = [
+                'type' => 'form',
+                'title' => 'Change Password',
```

Comment:
-  Can you please make sure to use `Yii::t` here.
- Only the word "Password" should be enough here.

Reply:
- done in https://github.com/humhub/humhub/pull/4598/commits/f72fefec9a0950e5d6f4724b0235c0aefcd658de
- it seems I copied that from tab "Account" which was not translatable either, I fixed that too there.